graylog2-web-interface/src/views/components/searchbar/queryinput/StyledAceEditor.jsx

```diff
@@ -18,12 +18,14 @@ import styled, { css } from 'styled-components';
 
 import AceEditor from './ace';
 
-const StyledAceEditor = styled(AceEditor).attrs(({ aceTheme, theme }) => ({
+const StyledAceEditor = styled(AceEditor).attrs(({ aceTheme, theme, $height }) => ({
   // NOTE: After setting the prop we need to swap them back so AceEditor uses the proper styles
   theme: aceTheme, /* stylelint-disable-line */
   $scTheme: theme,
-}))(({ $scTheme, disabled, value }) => css`
+  $height,
+}))(({ $scTheme, $height, disabled, value }) => css`
   &.ace-queryinput {
+    ${$height ? `height: ${$height}px !important` : ''};
```

Comment:
Is it possible to avoid using `!important`?

Reply:
We need to use `!Important` here, because the ace editor is also setting the height for this element by using the element style.